### PR TITLE
Services Notification URL Addition In Theming Context

### PR DESCRIPTION
**Pull request by asadmanzoor93**
**Description:** This PR adds Services Notification URL Addition In Theming Context

**JIRA:** 
https://edlyio.atlassian.net/browse/EDLY-1127

**Merge checklist:**

- [ ] All reviewers approved
- [ ] Commits are (reasonably) squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

**Note:** Please add screenshots for any viewable to change.

### CODE REVIEW

File: openedx/features/edly/context_processor.py [hunk 1]
```diff
@@ -38,6 +40,10 @@ def dynamic_theming_context(request):  # pylint: disable=unused-argument
         theming_context.update(
             {'edly_branding_config': configuration_helpers.get_dict('BRANDING', DEFAULT_BRANDING_DICT)}
         )
+        theming_context.update(
+            {'services_notifications_url': configuration_helpers.get_value('SERVICES_NOTIFICATIONS_URL',
```
Comment (zubair-arbi) [inline openedx/features/edly/context_processor.py]: I have only one concern, that this change is not related to theming.
Maybe we can have a new edly context processor `edly_app_context` where we can provide this value.

Comment (asadmanzoor93) [reply openedx/features/edly/context_processor.py]: These changes are required in edly themes and this is a custom context processer we added for our themes. Kindly suggest, should we write a separate context processor for this URL or not.

Comment (rehan99000) [reply openedx/features/edly/context_processor.py]: I think since it's a very small change, it's fine if it's done in this context processor. if a bigger change comes, we can make a separate context processor. Otherwise whatever zubair bhai thinks.

Comment (asadmanzoor93) [reply openedx/features/edly/context_processor.py]: As per discussion with @zubair-arbi we will add separate this context processor in a separate ticket.

Comment (asadmanzoor93) [reply openedx/features/edly/context_processor.py]: Feedback Implemented
